### ISSUE TITLE
Testing download SINAN

### ISSUE THREAD

**Issue by oranzani** [rfsaldanha/microdatasus#53]
Dear Rapha,
thank you for this package and all your work.

These days I was testing the new features to download files from SINAN, and I got some error when a variable has one format in Year XX and other format in Year YY. So, there was an error when `bind_rows` was trying to append them.

My suggestion would be two:

1 - Add the option in read.dbc to keep the variable "as is" instead of allowing to convert factors
(NB: this did not solve all problems, because after I have the same problem when it was character in year XX and integer on year YY)

2 - Changing all of them to character hard coding
(NB: the read.dbc -> foreign::read.dbf did not accept reading all as character function `col_types = cols(.default = 'c')`)

So, basically change this line of the function:

https://github.com/rfsaldanha/microdatasus/blob/b6317a7a87e3628fbce86c78e6fa4883458f58c7/R/fetch_datasus.R#L276

to:

`partial <- read.dbc::read.dbc(temp, as.is=TRUE) %>% mutate(across(everything(), as.character))`

Another option is to change to character only if it is not date format (since dates were unexpectedly doing well), but usually in my pipeline I have started to import/read almost everything as character upon first reading.

Hope it helps,
Thanks,
Otavio

**Comment by rfsaldanha** [comment]
Oi Otávio, obrigado pela atenção! Uma alteração dessa (todos como `character`) provavelmente iria afetar as outras funções do pacote. Vou pensar em algo mais específico. 

Como sempre, o tempo está curto e toda ajuda é bem vinda...


Abraços!
